storage/innobase/xtrabackup/src/xbstream.cc

```diff
@@ -668,46 +722,34 @@ static int mode_extract(int n_threads, int argc __attribute__((unused)),
       ds_set_pipe(ds_decrypt_zstd_ctxt, ds_decompress_zstd_ctxt);
     }
   }
+  data_threads = (extract_ctxt_t *)my_malloc(
+      PSI_NOT_INSTRUMENTED, sizeof(extract_ctxt_t) * (n_threads + 1),
+      MYF(MY_FAE));
+  for (int i = 0; i < n_threads; i++) {
+    data_threads[i].thread_id = i;
```

Comment:
We are now relaying this thread number to create a stream file. Please comment this somewhere.

Reply:
Done